x-pack/plugins/ml/common/types/locator.ts

```diff
@@ -250,3 +252,7 @@ export type MlUrlGeneratorState =
   | CalendarEditUrlState
   | FilterEditUrlState
   | MlGenericUrlState;
+
+export type MlLocatorParams = MlUrlGeneratorState & SerializableState;
```

Comment:
nit: `MlUrlGeneratorState` should be renamed here as well to be consistent with `MlLocator` usage

Reply:
Yes, will do.

Reply:
Done.